c7n/ctx.py

```diff
@@ -47,7 +47,9 @@ def __init__(self, session_factory, policy, options):
         self.sys_stats = None
 
         # A few tests patch on metrics flush
-        self.metrics = metrics_outputs.select(self.options.metrics_enabled, self)
+        # For backward compatibility, accept both 'metrics' and 'metrics_enabled' params
```

Comment:
can you add the pull request number here so we can date this, some day, we'll probably remove on the backwards compatibility.